moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -95,6 +95,10 @@ RobotStateDisplay::RobotStateDisplay() : Display(), update_state_(false), load_r
                                                      "Whether to display the collision representation of the robot.",
                                                      this, SLOT(changedEnableCollisionVisible()), this);
 
+  enable_attached_body_visible_ =
+      new rviz::BoolProperty("Attached Body Visible", true, "Whether to display the attached body.", this,
```

Comment:
For consistency the label should be "Attached Body Enabled". Or all three renamed to "Show ...". Not sure if renaming them would break config files?

Reply:
Yes, "Enabled" is more consistent, I just found it less descriptive. But I guess, it would be better to split the re-naming into a separate PR and use "Enabled" in this one.